### DISCUSSION TITLE
Individuelle Mail-Adressen/Benutzer im Kontaktformular

### DISCUSSION THREAD

**Discussion by Nachtmensch87** [codeispoetry/sunflower#144] [Ideen und Wünsche]
Es wäre schön, wenn im Sunflower-Kontaktformular individuelle Mail-Adressen und/oder in WP angelegte Nutzernamen als Empfänger hinterlegt werden könnten.

**Comment by codeispoetry** [comment]
Das ist umgesetzt. Man kann in den Sunflower-Einstellungen eine E-Mail-Adresse hinterlegen.

**Reply by Nachtmensch87** [reply]
@codeispoetry Wäre es möglich, es soweit zu individualisieren, dass man in den Block-Einstellungen die Empfänger-Adresse definieren kann? Aktuell kann ich nur eine Adresse hinterlegen. Manchmal wäre es aber hilfreich, wenn man auch für andere Dinge ein Kontaktformular hat, was aber in einem anderen Kontext eingesetzt wird und deshalb einen anderen Empfänger haben soll.

**Reply by FloJuh** [reply]
Duplicate of https://github.com/verdigado/sunflower/issues/657
